server/src/controllers/Events/resolver.ts

```diff
@@ -268,6 +268,18 @@ ${unsubscribe}
       subscribed: true, // TODO: even organizers may wish to opt out of emails
     };
 
+    const tagNames = data.tags
+      .map((tagName) => tagName.trim())
+      .filter((tagName) => tagName);
+    const eventTagsData: Prisma.tagsCreateManyInput[] = tagNames.map(
+      (tagName) => ({ name: tagName }),
+    );
+
+    await prisma.tags.createMany({ data: eventTagsData, skipDuplicates: true });
+    const tags = await prisma.tags.findMany({
+      where: { name: { in: tagNames } },
+    });
```

Comment:
It's a shame we can't get the ids as part of the query that creates them.  Oh well.

Reply:
I've digged some deeper into prisma relations, we might not need ids after all.

Alternative is removing duplicates before query and using nested `create`, with `connectOrCreate` for each tag. Per [documentation](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#create-a-single-record-and-multiple-related-records) this might be less performant and internally would use multiple queries, but that doesn't seem to be concern here right now.